examples/correlation_alignment_example.py

```diff
@@ -15,7 +15,7 @@
 rcParams["image.aspect"] = "auto"  # needed for sphinx gallery
 
 t2, td, vd, power, ph1, ph2 = s.symbols("t2 td vd power ph1 ph2")
-echo_time = 1e-3
+echo_time = 10e-3
```

Comment:
it wasn't clear to me whether or not the phasing was working b/c we the echo had been so assymetric, so I move the echo time out a bit just because it should give a pure red echo, which should be a better diagnostic

Reply:
so this actually made the phasing fall apart. It works better with the echo time being 1e-3 and as we increase the phasing from the hermitian falls apart.

Reply:
do we have to adjust the expressions to correlate with the newer echo time? (10 ms) to get this to work?

Reply:
if it falls apart for a longer echo time, that's not acceptable.  It could be you need to modify one of the kwargs for the hermitian phasing function, but then that should be obvious from the documentation.